packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -72,8 +72,8 @@ export class AtomicResultPrintableUri {
     });
   }
 
-  private getIndexOfEllipsis(parentsCount: number) {
-    return Math.min(parentsCount - 1, this.maxNumberOfParts);
```

Comment:
I'm not sure in what situation an ellipsis would be displayed and `parentsCount - 1` would be smaller than `this.maxNumberOfParts`...

Reply:
we only display an ellipsis when there is more than 1 element that it would unveil, like the breadbox does IIRC

if maxNumberOfParts = 3
`great-grand-parent > ... > parent > child` ❌ 
`great-grand-parent > grand-parent > parent > child` ✔️ 

now it's broken
![Screen Shot 2022-06-10 at 3 41 41 PM](https://user-images.githubusercontent.com/4923043/173138573-d6e57b9f-e968-4aa5-9c62-373dad17e673.png)
![Screen Shot 2022-06-10 at 3 41 44 PM](https://user-images.githubusercontent.com/4923043/173138570-7892bab0-d048-4561-8044-eb590a37236e.png)

Reply:
Actually it was already broken 😂 could you fix it in that PR?